### PR TITLE
Podman issues

### PR DESCRIPTION
**Pull request by johndeange**
## What changed

Changes needed to get `podman` working on my M3 Mac (`podman` is required by Flexion)

- Update `docker-compose.yml` files to be platform independent
- Separate `docker-compose.yml` profiles into separate files
- Use `--production` dependencies in FE `Dockerfile` (same as production)

## How to test

Compose still works and automated tests pass.

## Definition of Done Checklist
- [x] OESA: Code refactored for clarity
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [x] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [x] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [x] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- [x] Form validations updated

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ WORKDIR /home/app
 
 COPY --chown=app:app ./package.json ./bun.lockb /home/app/
 
-RUN bun install
+RUN bun install --production --frozen-lockfile
```
Comment (fpigeonjr) [inline frontend/Dockerfile:10]: 🤔 do we need the prod flag in localdev?

**Comment by fpigeonjr** [comment]
how did you get around the pre-commit hook as seen here:

https://github.com/HHS/OPRE-OPS/pull/3029/files#r1830201896

File: frontend/Dockerfile (hunk 1 above)
Comment (fpigeonjr) [reply frontend/Dockerfile:10]: maybe co-pilot agrees with me 🙃
image

Comment (fpigeonjr) [reply frontend/Dockerfile:10]: this is what the dev tools look like with prod flag 👎 
image

**Review by fpigeonjr — CHANGES_REQUESTED** [review]
Was able to get the app running with podman but don't think we want the prod flag for localdev since it effects the tooling:

`podman compose -f docker-compose.static.yml up --build`

File: frontend/Dockerfile (hunk 1 above)
Comment (fpigeonjr) [reply frontend/Dockerfile:10]: this is what they should look like 👍 
image

**Review by fpigeonjr — APPROVED** [review]

**Comment by fpigeonjr** [comment]
> Was able to get the app running with podman but don't think we want the prod flag for localdev since it effects the tooling:
> 
> `podman compose -f docker-compose.static.yml up --build`

disregard was running wrong command 🙃

**Review by maiyerlee — APPROVED** [review]

**Pull request merged**